src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/AndroidSdkResolver.cs

```diff
@@ -127,10 +127,12 @@ public static int GetMaxInstalledPlatform ()
 			foreach (var dir in Directory.EnumerateDirectories (Path.Combine (sdkPath, "platforms"))) {
 				int version;
 				string v = Path.GetFileName (dir).Replace ("android-", "");
+				Console.WriteLine ($"GetMaxInstalledPlatform: Parsing {v}");
 				if (!int.TryParse (v, out version))
 					continue;
 				if (version < maxInstalled)
 					continue;
+				Console.WriteLine ($"GetMaxInstalledPlatform: Setting maxInstalled to {version}");
```

Comment:
Maybe these should do `TestContext.Out.WriteLine()`, then we'd have the output in a file?

Reply:
the ProjectTools assembly doesn't reference NUnit. Which is why I used Console here.